### DISCUSSION TITLE
How to auto release to external TestFlight users?

### DISCUSSION THREAD

**Discussion by roderik** [fastlane/fastlane#17210] [Q&A]
I've tried the following

```
    upload_to_testflight(
      distribute_external: true, 
      groups: ["IdentiMint Beta"],
      skip_waiting_for_build_processing: true, 
      apple_id: "redacted", 
      uses_non_exempt_encryption: false
    )
```

`IdentiMint Beta` is the name of the external testing group which has 2 builds in there I manually added. Both are approved if that is relevant. The new builds are only shared with the `App Store Connect Users` group.

**Comment by jamesdixon** [comment]
@roderik were you able to figure this out? Running into the same issue

**Comment by taschik** [comment]
@roderik did you manage to find this one out? I'd like to ship a build to Testflight and add it automatically to once specific group.

**Reply by tcf909** [reply]
You have to set skip_waiting to false and supply release notes as a param. Be warned, waiting for release processing can take awhile so if your running in a pipeline the costs can add up.
